### DISCUSSION TITLE
Add option to track medication

### DISCUSSION THREAD

**Discussion by Ted-Corrrell** [babybuddy/babybuddy#621] [Ideas]
It would be great to have a way to track when a medication was administered. After a surgery, or dealing with an ear infection being able to track if a medication was given on time, or if enough time has passed that another dose can be administered. 
In my mind there would be a way to select a previously used medication as well as enter a new medication. It would also be beneficial to have a way to remove old medications, due to temporary prescriptions or a change in medications.

**Comment by cdubz** [comment]
There is a feature request for this over at #66. It has generated enough momentum over the years to get worked on and I think there are varying levels of expectations folks have around features so for the moment it is not planned. But I am open to further discussion/scope definition to help move it along.

**Comment by Ted-Corrrell** [comment]
Thanks for the reply. Got ya. I'm not great when it comes to finding things on github. I'll jump to #66 to keep things in one place.
